packages/sources/sportsdataio/src/sport/nba/endpoint/player-stats.ts

```diff
@@ -0,0 +1,173 @@
+  StatID: number
+  TeamID: number
+  PlayerID: number
+  SeasonType: number
+  Season: number
+  Name: string
+  Team: string
+  Position: string
+  Started: number
+  FanDuelSalary: number
+  DraftKingSalary: number
+  FantasyDuelSalary: number
+  YahooSalary: number | null
+  InjuryStatus: string | null
+  InjuryBodyPart: string | null
+  InjuryStartDate: string | null
+  InjuryNotes: string | null
+  FanDuelPosition: string
+  DraftKingPosition: string
+  YahooPosition: string | null
+  OpponentRank: number
+  OpponentPositionRank: number
+  GlobalTeamID: number
+  FantasyDraftSalary: number | null
+  FantasyDraftPosition: string | null
+  GameID: number
+  OpponentID: number
+  Opponent: string
+  Day: string
+  DateTime: string
+  HomeOrAway: string
+  isGameOver: true
+  GlobalGameID: number
+  GlobalOpponentID: number
+  Updated: string
+  Games: number
+  FantasyPoints: number
+  Minutes: number
+  Seconds: number
+  FieldGoalsMade: number
+  FieldGoalsAttempted: number
+  FieldGoalsPercentage: number
+  EffectiveFieldGoalsPercentage: number
+  TwoPointersMade: number
+  TwoPointersAttempted: number
+  TwoPointersPercentage: number
+  ThreePointersMade: number
+  ThreePointersAttempted: number
+  ThreePointersPercentage: number
+  FreeThrowsMade: number
+  FreeThrowsAttempted: number
+  FreeThrowPercentage: number
+  OffensiveRebounds: number
+  DefensiveRebounds: number
+  Rebounds: number
+  OffensiveReboundsPercentage: number
+  DefensiveReboundsPercenage: number
+  TotalReboundsPercentage: number
+  Assists: number
+  Steals: number
+  BlockedShots: number
+  Turnovers: number
+  PersonalFouls: number
+  Points: number
+  TrueShootingAttempts: number
+  TrueShootingPercentage: number
+  PlayerEfficiencyRating: number
+  AssistsPercentage: number
+  StealsPercentage: number
+  BlocksPercentage: number
+  TurnOversPercentage: number
+  UsageRatePercentage: number
+  FantasyPointsFanDuel: number
+  FantasyPointsDraftKings: number
+  FantasyPointsYahoo: number
+  PlusMinus: number
+  DoubleDoubles: number
+  TripleDoubles: number
+  FantasyPointsFantasyDraft: number
+  isClosed: boolean
+  LineupConfirmed: boolean
+  LineupStatus: string
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const { date, playerID } = validator.validated.data
+  const url = `/nba/stats/json/PlayerGameStatsByPlayer/${date}/${playerID || DEFAULT_PLAYER_ID}`
+
+  const params = {
+    key: config.nbaKey,
+  }
+
+  const options = { ...config.api, params, url }
+
+  const response = await Requester.request<ResponseSchema>(options)
+  return Requester.success(
+    jobRunID,
+    Requester.withResult(response, packResponse(response.data)),
+    config.verbose,
+  )
+}
+
+const packResponse = (response: ResponseSchema): string => {
+  const dataTypes = [
+    'uint16',
```

Comment:
I had to change this to `uint16` as `Season` did not fit into `uint8` specified in the spec https://www.notion.so/chainlink/LaMelo-Ball-NFT-ether-cards-0d4f5ca7f40a4b689daa799d9065e158.